server/middleware/measure.go

```diff
@@ -107,10 +130,18 @@ func measureStream() grpc.StreamServerInterceptor {
 			measurement.RecordDuration(metrics.RequestsErrorRespTime, measurement.GetRequestErrorTags(err))
 			ulog.E(err)
 			return err
+		} else {
+			measurement.CountOkForScope(metrics.RequestsOkCount, measurement.GetRequestOkTags())
+			_ = measurement.FinishTracing(wrapped.WrappedContext)
+			measurement.RecordDuration(metrics.RequestsRespTime, measurement.GetRequestOkTags())
+		}
+		if config.DefaultConfig.GlobalStatus.Enabled {
```

Comment:
instead of having this check here, can we move this check inside `RecordRequestToActiveChunk` i.e. if it is not enabled then `RecordRequestToActiveChunk` won't be doing anything.